resources/views/admin.blade.php

```diff
@@ -150,25 +186,35 @@ function deleteEmployee(id){
                 });
 
                 //Edit Employee
-                let $id = null
+                let id = null
                 $('#myTable tbody').on('click', '.btn-edit', function (e) {
-                    const $data =  table.row( $(this).parents('tr') ).data();
-                    $id  = $data.id
-                    $('#name').val($data.name)
-                    $('#position').val($data.position)
-                    $('#dob').val($data.dob)
-                    $('#email').val($data.email)
-                    $('#phone').val($data.phone)
-                    $('#address').val($data.address)
-
+                    id =  table.row( $(this).parents('tr') ).data().id;
+                    axios.get(`employee/${id}`)
+                    .then(function (response){
+                        const data = response.data[0];
+                        const selectOption= $("#selected-option")
+                        $('#name').val(data.name)
+                        $('#position').val(data.position)
+                        $('#dob').val(data.dob)
+                        $('#email').val(data.email)
+                        $('#phone').val(data.phone)
+                        $('#address').val(data.address)
+                        selectOption.text(data.company.name).val(data.company.id).data('branch' , data.company.branch)
```

Comment:
what motivates you to separate the element into variable?. other elements are directly mentioned and select only defined to variable?

Reply:
it was before a refactor of code when i had to select the Select Option twice...so i assigned it to a variable...after the refactoring i forgot to look into this...

ill address this and remove the variable @Raza9798

Reply:
@Raza9798  removed the variable....and directly mentioned the element